boards/st/stm32f411e_disco/stm32f411e_disco_stm32f411xe_B.overlay

```diff
@@ -29,3 +30,12 @@
 			  <&gpioe 5 GPIO_ACTIVE_HIGH>;
 	};
 };
+
+&spi1 {
+	/delete-node/ i3g4250d@0;
```

Comment:
This would deserve a comment

Reply:
Maybe only some revisions have the gyro populated. Unless there's a clear imbalance like 10 populated to 1 unpopulated, I would just only add it to the overlay of the revision that populates the part

Reply:
I believe that they all (STM32F411E-DISCO) had a gyro.  Just in revision D the chip was updated to I3G4250D

<img width="1193" height="549" alt="image" src="https://github.com/user-attachments/assets/696945cd-1695-48af-9b70-b8f6d15b2b0c" />